internal/postprocessor/main.go

```diff
@@ -37,16 +37,19 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-var apiNameOwlBotScope = "[REPLACEME]"
-
-// hashFromLinePattern grabs the hash from the end of a github commit URL
-var hashFromLinePattern = regexp.MustCompile(`.*/(?P<hash>[a-zA-Z0-9]*).*`)
-
-// firstPartTitlePattern grabs the existing commit title before the ': [REPLACEME]'
-var firstPartTitlePattern = regexp.MustCompile(`(?P<titleFirstPart>)(\: *\` + apiNameOwlBotScope + `)(.*)`)
+const (
+	owlBotBranchPrefix = "owl-bot-"
```

Comment:
Does this replace the need for the branchPrefix flag, or should we change the flag to match this prefix?

Reply:
Got it! Thanks